### PR TITLE
translations PR 20171002

### PR DESCRIPTION
**Pull request by enahum**
Ideally this PR should be review by community members for each language.

Note: In the event that you see an english translation instead of the desired language is because this PR was submitted before the translation was made on the Mattermost Translation Server and probably will be submitted for the next PR.

The corresponding webapp PR is here: https://github.com/mattermost/mattermost-webapp/pull/99

### CODE REVIEW

File: i18n/pt-BR.json [hunk 1]
```diff
@@ -3688,6 +3688,10 @@
     "translation": "Exportação da conformidade para tarefa '{{.JobName}}' iniciada no '{{.FilePath}}'"
   },
   {
+    "id": "ent.data_retention.generic.license.error",
```
Comment (cpanato) [inline i18n/pt-BR.json]: Did not find this string in the translation server

**Comment by cpanato** [comment]
`pt-br`lgtm except for the comment I made in the PR

**Comment by wget** [comment]
Again, no change for french this week. `fr` LGTM.

File: i18n/pt-BR.json (hunk 1 above)
Comment (jasonblais) [reply i18n/pt-BR.json]: @cpanato I think the PR that added just got merged. https://github.com/mattermost/mattermost-server/pull/7539

Should be in the Translation server after the next sync, which is in a couple of hours

**Comment by kaakaa** [comment]
`ja` LGTM!

**Pull request merged**